File: spring-cloud-bus/bus-commons/src/main/java/com/livk/bus/LivkBusConfiguration.java
package com.livk.bus;

import com.livk.auto.service.annotation.SpringAutoService;
import com.livk.bus.event.LivkBusEvent;
import org.springframework.boot.autoconfigure.AutoConfiguration;
import org.springframework.cloud.bus.jackson.RemoteApplicationEventScan;

/**
 * <p>
 * LivkBusConfiguration
 * </p>
 *
 * @author livk
 * @date 2021/11/1
 */
@AutoConfiguration
@SpringAutoService
@RemoteApplicationEventScan(basePackageClasses = LivkBusEvent.class)
public class LivkBusConfiguration {

}

File: spring-cloud-bus/bus-commons/src/main/kotlin/com/livk/bus/LivkBusConfiguration.kt
package com.livk.bus

import com.livk.auto.service.annotation.SpringAutoService
import com.livk.bus.event.LivkBusEvent
import org.springframework.boot.autoconfigure.AutoConfiguration
import org.springframework.cloud.bus.jackson.RemoteApplicationEventScan

/**
 *
 * @author livk
 */
@AutoConfiguration
@SpringAutoService
@RemoteApplicationEventScan(basePackageClasses = [LivkBusEvent::class])
open class LivkBusConfiguration
